src/content/en/2022/performance.md

```diff
@@ -1,22 +1,989 @@
+
+Thankfully, there is a way to measure page load speed automatically! We know the browser is in charge of loading a page, and it goes through a checklist of steps each time. Depending on which step the browser is on, we can tell how far along the site is in the page load process. Conveniently, a [number of performance timeline APIs](https://developer.mozilla.org/en-US/docs/Web/API/Performance) are used to fire off timestamps when the browser gets to certain page load steps. One such API, the [User Timing API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API), even allows engineers to add custom timestamps for their specific needs. These timestamps become the metrics used across the industry to decipher how performant a page is.
+
+However, these metrics are only our best guess at how to gauge user experience. For example, just because the browser fired an event that an element has been painted onto the screen, does that always mean it was visible to the user at that time? Additionally, as the industry grew, more and more metrics showed up while some became deprecated. It can be complicated to know where to start and understand what performance metrics are telling us about our users, especially for folks newer to the field.
+
+Enter <a hreflang="en" href="https://web.dev/vitals/">CWV</a>, three metrics to indicate overall performance health. Each metric covers an important area of user experience: loading, interactivity, and visual stability. The public <a hreflang="en" href="https://developer.chrome.com/docs/crux/">Chrome UX Report</a> (CrUX) dataset is Chrome's view of how millions of websites are performing on CWV. There's zero setup on the developer's part; Chrome automatically collects and publishes data from <a hreflang="en" href="https://developer.chrome.com/docs/crux/methodology/#eligibility">eligible</a> users and websites. Using CrUX data, we're able to get insights into the web's performance and even track historical data.
+
+CWV offers a clearly defined approach to measuring how user's actually experience performance—a first for the industry. Are CWV the answer to helping the web become more performant? It's been a little over two years since launch, so this chapter examines where the web is currently with CWV and a look into the future.
+
```

Comment:
I think we should add a caveat about Rick's job at the env of the Introduction, similar to [how we've done in the other chapters](https://almanac.httparchive.org/en/2022/jamstack#:~:text=Disclosure%3A%20the%20two%20authors%20of%20this%20report%20were%20Netlify%20employees.%20Netlify%20invented%20the%20term%20Jamstack%20and%20owns%20Jamstack.org.%20This%20report%20and%20the%20underlying%20analysis%20were%20reviewed%20and%20approved%20by%20others%20not%20affiliated%20with%20Netlify.).

Reply:
Not sure if we want to name Rick or not here, the example didn't mention specific names but it sounds like all authors were included in the disclaimer. I'll leave that up to y'all!

Reply:
Rephrasing slightly but LGTM